src/logic/function/basic.lean

```diff
@@ -5,6 +5,7 @@ Authors: Johannes Hölzl, Mario Carneiro
 -/
 import logic.basic
 import data.option.defs
+import tactic.basic
```

Comment:
Can you make do without the injection tactic here?

Reply:
`injection` is in core lean, I think it's the `split_ifs` that's the issue.